Model/Cc.php

```diff
@@ -601,10 +601,7 @@ protected function getConsumerInfo($order)
      */
     private function getCallbackUrl($order)
```

Comment:
would it be a hassle to also rename this getter? 

or is this an expected function name that we shouldn't change? 
(ex expected by Magento to look like this, in which case should we adjust our terminology from notification to callback in `rest/default/V1/reach/notification`?)

Reply:
This method is only ever called in this class so we could change it to whatever we like

I also tested it locally and confirmed that changing it doesn't have any negative impacts. 

What do you think of `getNotifyUrl`? Or would `getNotificationUrl` be more fitting?